new/components/basic/form/BasicTextInput.tsx

```diff
@@ -1,26 +1,26 @@
 "use client"
-import {InputHTMLAttributes} from "react";
-import {View, Text, TextInput, StyleSheet} from "react-native";
-import {TextInputProps} from "react-native/Libraries/Components/TextInput/TextInput";
+import { InputHTMLAttributes } from "react";
+import { View, Text, TextInput, StyleSheet } from "react-native";
 
 interface SimpleInputProps extends InputHTMLAttributes<HTMLInputElement> {
-    label ?: string,
-    autoComplete ?: string
+    label?: string,
+    validationList?: ((input: any) => boolean)[],
```

Comment:
avoid using a validation List. instead , the component should looks like
`<BasicTextInput required pattern="[a-zA-Z]" max="30"/>` which is a required field, accept letter only and 30 letters maximum.

The props interface should include all the supported validation.